huxley/www/static/js/huxley/components/AdvisorRosterView.js

```diff
@@ -109,6 +114,51 @@ var AdvisorRosterView = React.createClass({
     }.bind(this));
   },
 
+  _addDelegatePressed: function() {
+    return (
+      <form>
+        Name: <input type="text" placeholder="Name" valueLink={this.linkState('name')} /><br>
+        Email: <input type="text" placeholder="Email" valueLink={this.linkState('email')}/><br>
+        Summary: <input type="text" placeholder="Summary" valueLink={this.linkState('summary')}/><br>
+        <input type="submit" value="Submit" onclick={this._handleSubmit} />
+      </form>
+    )
+  },
+
+  _handleSubmit: function(data) {
+    this.setState({loading: true});
+    $.ajax({
+      type: 'POST',
+      url: '/api/delegate',
+      data: {
+        name: this.state.name,
+        email: this.state.email,
+        summary: this.state.summary
```

Comment:
Advisors shouldn't be able to submit a summary. The summary should be read only.